python/cuml/test/test_train_test_split.py

```diff
@@ -385,18 +385,41 @@ def test_stratified_binary_classification():
     with pytest.raises(ValueError):
         train_test_split(X, y,
                          train_size=0.75,
-                         stratify=True,
-                         shuffle=True)
+                         stratify=y,
+                         shuffle=y)
```

Comment:
Just want to confirm this is intended to error because of shuffle having a bad value?

Reply:
No, that was a mistake, good catch. I've fixed it now.